core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorTestBase.java

```diff
@@ -134,6 +150,9 @@ protected void assertTempBlockMeta(Allocator allocator, BlockStoreLocation locat
     }
   }
 
+  /**
+   * @return the block metadata managerView
```

Comment:
the javadoc doesn't say anything new that the signature doesn't already say, so we can omit this javadoc